sdk/core/core-http/src/policies/logPolicy.ts

```diff
@@ -13,6 +13,9 @@ import { Debugger } from "@azure/logger";
 import { logger as coreLogger } from "../log";
 import { Sanitizer } from "../util/sanitizer";
 
+/**
+ * Allows configuring what header values should be logged, what query parameters should be logged, and also passing a custom logger.
+ */
 export interface LogPolicyOptions {
   /**
    * Header names whose values will be logged when logging is enabled. Defaults to
```

Comment:
I noticed that the list of defaults is no longer accurate. The current list is available in the sanitizer: https://github.com/Azure/azure-sdk-for-js/blob/ed5afab0edd38be8cc09dfb40b0cbb847894077b/sdk/core/core-http/src/util/sanitizer.ts#L25